integrationtest/backend/elasticsearch/src/test/java/org/hibernate/search/integrationtest/backend/elasticsearch/schema/management/ElasticsearchIndexSchemaManagerOperation.java

```diff
@@ -22,42 +23,42 @@ enum ElasticsearchIndexSchemaManagerOperation {
 
 	CREATE_IF_MISSING {
 		@Override
-		public CompletableFuture<?> apply(IndexSchemaManager schemaManager) {
-			return schemaManager.createIfMissing();
+		public CompletableFuture<?> apply(IndexSchemaManager schemaManager, OperationSubmitter operationSubmitter) {
+			return schemaManager.createIfMissing( operationSubmitter );
 		}
 	},
 	DROP_AND_CREATE {
 		@Override
-		public CompletableFuture<?> apply(IndexSchemaManager schemaManager) {
-			return schemaManager.dropAndCreate();
+		public CompletableFuture<?> apply(IndexSchemaManager schemaManager, OperationSubmitter operationSubmitter) {
+			return schemaManager.dropAndCreate( operationSubmitter );
 		}
 	},
 	CREATE_OR_VALIDATE {
 		@Override
-		public CompletableFuture<?> apply(IndexSchemaManager schemaManager) {
-			return schemaManager.createOrValidate( new StubUnusedContextualFailureCollector() );
+		public CompletableFuture<?> apply(IndexSchemaManager schemaManager, OperationSubmitter operationSubmitter) {
+			return schemaManager.createOrValidate( new StubUnusedContextualFailureCollector(), operationSubmitter );
 		}
 	},
 	CREATE_OR_UPDATE {
 		@Override
-		public CompletableFuture<?> apply(IndexSchemaManager schemaManager) {
-			return schemaManager.createOrUpdate();
+		public CompletableFuture<?> apply(IndexSchemaManager schemaManager, OperationSubmitter operationSubmitter) {
+			return schemaManager.createOrUpdate( operationSubmitter );
 		}
 	},
 	DROP_IF_EXISTING {
 		@Override
-		public CompletableFuture<?> apply(IndexSchemaManager schemaManager) {
-			return schemaManager.dropIfExisting();
+		public CompletableFuture<?> apply(IndexSchemaManager schemaManager, OperationSubmitter operationSubmitter) {
+			return schemaManager.dropIfExisting( operationSubmitter );
 		}
 	},
 	VALIDATE {
 		@Override
-		public CompletableFuture<?> apply(IndexSchemaManager schemaManager) {
-			return schemaManager.validate( new StubUnusedContextualFailureCollector() );
+		public CompletableFuture<?> apply(IndexSchemaManager schemaManager, OperationSubmitter operationSubmitter) {
+			return schemaManager.validate( new StubUnusedContextualFailureCollector(), operationSubmitter );
 		}
 	};
 
-	public abstract CompletableFuture<?> apply(IndexSchemaManager schemaManager);
+	public abstract CompletableFuture<?> apply(IndexSchemaManager schemaManager, OperationSubmitter operationSubmitter);
```

Comment:
As far as I can see, you always pass `BLOCKING` to this method. And it's in test code. So... Maybe just hardcode `BLOCKING` in the implementations, and don't bother changing a million tests? :)

I mean if you want to test both submitters in some tests, then go ahead, but I'm not sure that would be a great use of your time :)